patterns/hidden-search.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * Title: Search
+ * Slug: twentytwentyfive/hidden-search
+ * Inserter: no
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+
+<!-- wp:search {"label":"<?php esc_html_e( 'Search', 'twentytwentyfive' ); ?>","showLabel":false,"buttonText":<?php echo esc_attr_x( 'Search', 'button label', 'twentytwentyfive' ); ?>","style":{"border":{"radius":"50px"}},"fontSize":"medium"} /-->
```

Comment:
There's a missing quote at the start of the `ButtonText` value.

Reply:
Fixed below: